cmd/brew-setup-nginx-conf

```diff
@@ -79,6 +79,15 @@ if `readlink /etc/resolver 2>/dev/null`.chomp != "/usr/local/etc/resolver"
   end
 end
 
+if File.exist? "/etc/pf.anchors/dev.strap"
+  unless system "sudo -n true >/dev/null"
+    puts "Asking for your password to setup launch_socket_server:"
+  end
+  system "sudo rm /etc/pf.anchors/dev.strap"
+  system "brew install launch_socket_server"
+  system "brew services restart launch_socket_server"
```

Comment:
Also probably want to rework `started_nginx` below to be `brewfile_installs` or something and then restart both `nginx` and `launch_socket_server` and check `launch_socket_server`'s status with e.g. `unless system "sudo brew services restart launch_socket_server >/dev/null"`. Maybe also worth adding a `sudo -k` at the end of this file, too (which I'd forgotten to do previously).